app/controllers/notes_controller.rb

```diff
@@ -22,7 +22,7 @@ def places
       .references(:term_data)
       .where('term_data.name = ?', 'chapter')
       .group('node.nid')
-      .order('max(node_revisions.timestamp) DESC, node.nid')
+      .order(Arel.sql('max(node_revisions.timestamp) DESC, node.nid'))
```

Comment:
Whoa, what's this? How does it work?

Reply:
It's basically removing the deprecation warning : 
`DEPRECATION WARNING: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "max(node_revisions.timestamp) DESC, node.nid". Non-attribute arguments will be disallowed in Rails 6.0. This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql(). (called from places at /app/app/controllers/notes_controller.rb:25)`

and as they have written in the warning itself that `Known-safe values can be passed by wrapping them in Arel.sql()`. So, I followed them.